app/models/comment.rb

```diff
@@ -1,4 +1,9 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+  # , counter_cache: :comments_counter
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] Kindly remove commented code.

- **[OPTIONAL]** I would suggest you add an active record callback for the `update_comments_counter`